econet_hvac_furnace.yaml

```diff
@@ -10,9 +10,11 @@ packages:
 econet:
   request_mod_addresses:
     5: 448
+    6: 448
```

Comment:
In a separate PR, have a substitution for this address and use it in all places. It will improve readability since this address will have a name and it will allow users to easily override it in one place.